cli/src/code_coverage/prvsnr-coverage.py

```diff
@@ -32,7 +32,7 @@ def install_coverage():
         "yum install -y gcc cpp python3-devel"
     )
     _run(
-        "pip3 install -U api/python"
+        "pip3 install -U api/python[test]"
```

Comment:
please compare `test` extra in setup.py with test-requirements.txt and add missed dependencies in the former. So the next step (`pip3 install -r test-requirements.txt`) can be removed.

Reply:
Done !